azerothcore/azerothcore-wotlk
# fix(Scripts/MoltenCore): Ensure Majordomo minions that completely des…

…pawned respawn on wipes



## Changes Proposed:
-  Store spawn data for Majordomo adds and respawn the ones completely despawned on wipes.
-  This happens because the instance implementation of minion creatures can't respawn them if they no longer exist in world (despawned after corpse decay)

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Tested in-game.
- 


## How to Test the Changes:


1. Engage Majordomo, kill his adds, except one foc
2. Wait about 2-3 minutes, so the add corpses despawn
3. Reset encounter, notice all of them respawn (even the ones that were completely despawned)

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
